WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
> I'm saying the 1.23 part is a string, since JSON is one big string, even though this is the JSON encoding of a number.

I understand that, but every implementation of JSON parser/generator that I've dealt with requires you to pass a float in whatever language you're dealing with to generate such a JSON string. For example, in javascript:

```javascript
~ @kaladin> node
> JSON.stringify({ seconds: 1.23 })
'{"seconds":1.23}'
```

This is why I say that you'd have to customize the JSON parser to deal with WCIF properly